video/out/w32_common.c

```diff
@@ -1768,6 +1768,11 @@ static int gui_thread_control(struct vo_w32_state *w32, int request, void *arg)
         update_display_info(w32);
         *(double*) arg = w32->display_fps;
         return VO_TRUE;
+    case VOCTRL_GET_DISPLAY_RES: ;
+        RECT r = get_screen_area(w32);
```

Comment:
Nice. I was about to comment that `get_screen_area` is likely more aligned with the docs than `get_working_area`, but you changed it on your own :)